.github/workflows/ruby.yml

```diff
@@ -23,8 +24,19 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: ['3.0', '3.1', '3.2']
-        activerecord: ['6.0', '6.1', '7.0', '7.1']
+        ruby-version: ['3.0', '3.1', '3.2', '3.3']
```

Comment:
actually, since now we're already in Rails 7.0 and Ruby 3.1, we only need the testing for
- Rails 7.0, 7.1, 7.2
- Ruby 3.1, 3.2, 3.3

The inclusion of Rails 6.0, 6.1 and also Ruby 3.0 is okay, just that it's redundant for our codebase (this library is only for our internal usage anyway)

Reply:
Removed the rails 6 configurations, leaving behind rails 7.0, 7.1, 7.2 and Ruby 3.0, 3.1, 3.2, 3.3.